pandas/core/indexes/datetimes.py

```diff
@@ -1747,7 +1747,33 @@ def freq(self, value):
     is_year_end = _field_accessor(
         'is_year_end',
         'is_year_end',
-        "Logical indicating if last day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the last day of the
+        year.
+
+        Returns
+        -------
+        is_year_end : Series of boolean.
+
+        See Also
+        --------
+        is_year_start : Return a boolean indicating whether the date is the
+            first day of the year.
+
+        Examples
+        --------
+        >>> dates = pd.Series(pd.date_range("2017-12-30", periods=3))
+        >>> dates
+        0   2017-12-30
+        1   2017-12-31
+        2   2018-01-01
+        dtype: datetime64[ns]
+        >>> dates.dt.is_year_end
+        0    False
+        1     True
+        2    False
+        dtype: bool
+        """)
```

Comment:
Could you add an example with just a DatetimeIndex? Can be the same values, and don't have to show the values again,

```
>>> pd.date_range('2017-12-30", periods=3).is_year_end
```

and show the output